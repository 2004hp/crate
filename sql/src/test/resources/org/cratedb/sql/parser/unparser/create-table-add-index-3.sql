create table t1 (c1 int, c2 double, c3 int, index "ft" using fulltext(c1, c2, c3) with (analyzer='german'))