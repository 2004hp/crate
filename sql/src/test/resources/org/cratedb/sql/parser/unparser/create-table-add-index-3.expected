CREATE TABLE t1(c1 INTEGER, c2 DOUBLE, c3 INTEGER, INDEX ft USING fulltext(c1, c2, c3) WITH ("analyzer"='german'))